test/browser/browser.sh

```diff
@@ -56,9 +56,18 @@ echo core > /proc/sys/kernel/core_pattern
 
 # grab a few images to play with; tests run offline, so they cannot download images
 podman rmi --all
-podman pull quay.io/libpod/busybox
-podman pull quay.io/libpod/alpine
-podman pull quay.io/cockpit/registry:2
+
+# use images which support multiple architectures
+podman pull quay.io/prometheus/busybox
+podman pull quay.io/jitesoft/alpine
+podman pull quay.io/libpod/registry:2.8
+# the tests expect the image to be called differently, so re-tag them
+podman tag quay.io/prometheus/busybox quay.io/libpod/busybox
+podman rmi quay.io/prometheus/busybox
+podman tag quay.io/jitesoft/alpine quay.io/libpod/alpine
+podman rmi quay.io/jitesoft/alpine
+podman tag quay.io/libpod/registry:2.8 quay.io/cockpit/registry:2
+podman rmi quay.io/libpod/registry:2.8
```

Comment:
Ideally we would do this in `images/scripts/lib/podman-images.setup` in cockpit/bots.

Reply:
Yes, of course -- my plan is to migrate our VM images to these new images as well. But it's painful to migrate them all at once.

Reply:
I.e. my current idea was to land this, then have a bots PR which updates to these images and e.g. rebuild the two Debians (and let the rest happen naturally). Then we can update test/browser/browser.sh to call that script.

The bit that still sucks is how we are going to switch to the real new image names. Renaming them forever sounds a bit sad. Any ideas?

Reply:
I.e. we can't pull the old and new images, as that messes up c-podman's expectations which images exist.

Reply:
@jelly and I talked and we have a plan. This will temporarily look ugly and duplicated here, but once this PR lands, the path is free for refreshing our VM images with the new container images. See the commit message for details.

Reply:
See https://github.com/cockpit-project/bots/pull/4585